src/entities/task/ui.tsx

```diff
@@ -1,16 +1,66 @@
-import { useSelector } from "react-redux";
+import { SyntheticEvent, useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 
-import { selectTasks, Task } from "./model";
+import { deleteTask, selectAll, Task, updateTask } from "./model";
 
-const TaskItem = ({ title }: Omit<Task, "completed" | "id">) => <div>{title}</div>;
+const TaskItem = ({ title, id }: Omit<Task, "completed">) => {
+    const dispatch = useDispatch();
+    const [text, setText] = useState("");
+    const [isEditing, setIsEditing] = useState(false);
+    const handleClick = (id: string) => {
+        dispatch(deleteTask(id));
+    };
+
+    const handleChange = (e: SyntheticEvent) => {
```

Comment:
`e: ChangeEvent<HTMLInputElement>`

```
    const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
        setText(e.target.value);
    };
  ```
  
  // Nitpick. Use `evt` instead of `e`